components/application-broker/internal/broker/provision.go

```diff
@@ -261,6 +315,86 @@ func (svc *ProvisionService) ensureEaUpdate(appID, ns string, si *v1beta1.Servic
 	return nil
 }
 
+// enableDefaultKnativeBroker enables the Knative Eventing default broker for the given namespace
+// by adding the proper label to the namespace.
+func (svc *ProvisionService) enableDefaultKnativeBroker(ns internal.Namespace) error {
+	// get the namespace
+	namespace, err := svc.knClient.GetNamespace(string(ns))
+	if err != nil {
+		svc.log.Printf("error getting namespace: [%s] [%v]", ns, err)
```

Comment:
Shouldn't be `svc.log.Errorf()`

Reply:
True. Btw, there is a ticket to deprovision the broker and the label on the namespace, see this ticket for more details: https://github.com/kyma-project/kyma/issues/6342

Reply:
Follow up PR: https://github.com/kyma-project/kyma/pull/6728/files